app/templates/views/user-profile.html

```diff
@@ -63,6 +63,17 @@ <h1 class="font-body-2xl margin-bottom-3">User profile</h1>
           )
         }}
       {% endcall %}
+      IS IT HERE???? {{ current_user.preferred_timezone }}
```

Comment:
It looks like this is a debug statement that got left in.

```suggestion
```

Reply:
This actually was here.  I just removed it.  There is a bug in GitHub where sometimes reviews don't update to the latest pushed commit, but it didn't happen this time.  This time was developer error.

As for handling the lack of the default "US/Eastern", that was addressed in a migration in api project separately.

Reply:
Awesome, thank you again!